README.md

```diff
@@ -1,94 +1,157 @@
-# The ops library
+# The `ops` library
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+The `ops` library is a Python framework for developing and testing Kubernetes and machine charms. While charms can be written in any language, `ops` defines the latest standard, and all new charms are encouraged to use Python with `ops`. The library is an official component of the Charm SDK, itself a part of [the Juju universe](https://juju.is/).
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
+> - `ops` is  [available on PyPI](https://pypi.org/project/ops/).
+> - The latest version of `ops` requires Python 3.8 or above.
```

Comment:
I don't feel super strongly about this, but I find the left border to just be noise here. If you feel it should be there then feel feel to ignore this comment.

```suggestion
- `ops` is  [available on PyPI](https://pypi.org/project/ops/).
- The latest version of `ops` requires Python 3.8 or above.
```

Reply:
I don't mind badges when used in moderation, although a lot of badges rely on third-party services and I don't like having that as a 'dependency' for a README (plus the badges work fine when you're reading this on Github but if you've checked out the repo and are reading it there not so much). It's definitely something that could be considered in the future, though.